v3/src/data-model/attribute.ts

```diff
@@ -31,7 +31,7 @@
   more like `math.js` but that decision can be left for another day.
  */
 
-import { Instance, SnapshotIn, types } from "mobx-state-tree"
+import {IAnyType, Instance, SnapshotIn, types} from "mobx-state-tree"
```

Comment:
```suggestion
import {Instance, SnapshotIn, types} from "mobx-state-tree"
```
Apparently, `IAnyType` is unused.